An error on babel6

I found an error on babel6 project. Please see the example project.

https://github.com/zakuro9715/espower-babel-babel6-error-example

On babel5

```
$ npm run test

> @ test /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel5
> mocha --compilers js:espower-babel/guess test/add.test.js



  add
    ✓ should returns result of A plus B


  1 passing (8ms)
```

On babel6

```
$ npm run test

> @ test /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6
> mocha --compilers js:espower-babel/guess test/add.test.js

/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/logger.js:58
    throw new Constructor(this._buildMessage(msg));
    ^

ReferenceError: [BABEL] /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/test/add.test.js: Unknown option: direct.presets
    at Logger.error (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
    at OptionManager.mergeOptions (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
    at OptionManager.init (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/options/option-manager.js:216:10)
    at File.initOptions (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/index.js:147:75)
    at new File (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/index.js:137:22)
    at Pipeline.transform (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at Object.extensions.(anonymous function) [as .js] (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/espower-babel/index.js:56:28)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/lib/mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/lib/mocha.js:213:14)
    at Mocha.run (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/lib/mocha.js:453:10)
    at Object.<anonymous> (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/bin/_mocha:393:18)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3

npm ERR! Linux 4.2.0-1-amd64
npm ERR! argv "/home/zakuro/.nvm/versions/node/v5.0.0/bin/node" "/home/zakuro/.nvm/versions/node/v5.0.0/bin/npm" "run" "test"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
npm ERR! @ test: `mocha --compilers js:espower-babel/guess test/add.test.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ test script 'mocha --compilers js:espower-babel/guess test/add.test.js'.
npm ERR! This is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha --compilers js:espower-babel/guess test/add.test.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/npm-debug.log
```

--- a/.babelrc
+++ b/.babelrc
@@ -1,3 +1,4 @@
 {
-    "stage": 0
+  "presets": ["es2015", "stage-3"],
+  "plugins": ["transform-es2015-modules-commonjs"]
 }

--- a/README.md
+++ b/README.md
@@ -174,7 +174,7 @@ require('espower-babel')({
 
 **Caution**:
  
-Babel 6 no transform by default.
+Babel 6 has not transform by default.
 It means that you must set babel config by `.babelrc` file or `babelrc` option.
 
 ### Transform all files with Babel **by default**

--- a/README.md
+++ b/README.md
@@ -168,7 +168,7 @@ e.g.)
 ```js
 require('espower-babel')({
     babelrc: {
-        "presets": ["es2015", "stage-3"],
+        "presets": ["es2015"],
         "plugins": ["transform-es2015-modules-commonjs"]
     }
 })

--- a/README.md
+++ b/README.md
@@ -89,7 +89,7 @@ See the power-assert output appears!
 
 ## HOW TO USE
 
-### Zero-config mode
+### Zero-config mode(for testing)
 
 If your tests are located on `'test/**/*.js'`, just run mocha with `--compilers js:espower-babel/guess`
 
@@ -161,15 +161,21 @@ Babel has many transform options.
 
 `espower-babel` read `${cwd}/.babelrc` if exists.
 
-also can manually configure babel transform options.
+Also, you can manually configure babel transform options.
 
 ```js
 require('espower-babel')({
     babelrc: {
-        stage: 0
+        "presets": ["es2015", "stage-3"],
+        "plugins": ["transform-es2015-modules-commonjs"]
     }
 })
 ```
+
+**Caution**:
+ 
+Babel 6 no transform by default.
+It means that you must set babel config by `.babelrc` file or `babelrc` option.
 
 ### Transform all files with Babel **by default**
 

--- a/README.md
+++ b/README.md
@@ -163,6 +163,8 @@ Babel has many transform options.
 
 Also, you can manually configure babel transform options.
 
+e.g.)
+
 ```js
 require('espower-babel')({
     babelrc: {
@@ -180,6 +182,8 @@ It means that you must set babel config by `.babelrc` file or `babelrc` option.
 ### Transform all files with Babel **by default**
 
 Do limit transform files by setting `babelrc`
+
+e.g.)
 
 ```js
 require('espower-babel')({

--- a/README.md
+++ b/README.md
@@ -174,7 +174,7 @@ require('espower-babel')({
 
 **Caution**:
  
-Babel 6 has not transform by default.
+Babel 6 does not transform your code by default. 
 It means that you must set babel config by `.babelrc` file or `babelrc` option.
 
 ### Transform all files with Babel **by default**

--- a/package.json
+++ b/package.json
@@ -33,15 +33,19 @@
     "url": "https://github.com/power-assert-js/espower-babel/issues"
   },
   "dependencies": {
-    "babel-core": "^5.0",
-    "babel-plugin-espower": "^1.0.0",
-    "minimatch": "^2.0.1",
-    "source-map-support": "^0.2.10",
+    "babel-core": "^6.0.0",
+    "babel-plugin-espower": "^2.0.0",
+    "minimatch": "^3.0.0",
+    "source-map-support": "^0.3.3",
     "xtend": "^4.0.0"
   },
   "devDependencies": {
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.1.18",
+    "babel-polyfill": "^6.1.19",
+    "babel-preset-es2015": "^6.1.18",
+    "babel-preset-stage-3": "^6.1.18",
     "expect.js": "^0.3.1",
     "mocha": "^2.1.0",
-    "power-assert": "^0.11.0"
+    "power-assert": "^1.1.0"
   }
 }

--- a/test/tobe_instrumented/es7_test.js
+++ b/test/tobe_instrumented/es7_test.js
@@ -1,11 +1,10 @@
-require("babel-core/polyfill")
+require("babel-polyfill")
 
 let assert = require('power-assert')
 
 describe("ES7 async/await", ()=>{
   it("works", async()=>{
-    let ok = await Promise.resolve("OK")
 
-    assert(`${ok}` === "OK")
+    assert(await Promise.resolve("OK") === "OK")
   })
 })

--- a/test_loader/espower-traceur-loader.js
+++ b/test_loader/espower-traceur-loader.js
@@ -20,6 +20,7 @@ require('..')({
         ]
     },
     babelrc: {
-        stage: 0
+        "presets": ["es2015", "stage-3"],
+        "plugins": ["transform-es2015-modules-commonjs"]
     }
 });

